tests/performance_tests/README.md

```diff
@@ -0,0 +1,605 @@
+
+The `performance_test_basic.py` script performs a single basic performance test that targets a configurable TPS target and, if successful, reports statistics on perfomance metrics measured during the test.  It configures and launches a blockchain test environment, creates wallets and accounts for testing, configures and launches transaction generators for creating specific transaction load in the ecosystem.  Finally it analyzes the performance of the system under the configuration through log analysis and chain queries and produces a [Performance Test Basic Report](#performance-test-basic).
+
+The `launch_generators.py` script provides a means to easily calculate and spawn the number of transactions generator instances to generate a given target TPS, distributing generation load between the instances in a fair manner such that the aggregate load meets the requested test load.
+
+The `log_reader.py` script is used primarily to analyze `nodeos` log files to glean information about generated blocks and transactions within those blocks after a test has concluded.  This information is used to produce the performance test report. In similar fashion, `read_log_data.py` allows for recreating a report from the configuration and log files without needing to rerun the test.
+
+## Prerequisites
+
+Please refer to [Leap: Software Installation](https://github.com/AntelopeIO/leap#software-installation)
+
+## Steps
+
+1. Install Leap. For complete instructions on obtaining compiled binaries or building from source please refer to [Leap: Building From Source](https://github.com/AntelopeIO/leap#building-from-source)
+2. Run Tests
+    1. Full Performance Test Run:
+        ``` bash
+        ./build/tests/performance_tests/performance_test.py
+        ```
+    2. Single Performance Test Basic Run:
+        ```bash
+        ./build/tests/performance_tests/performance_test_basic.py
+        ```
+3. Collect Results - If specifying `--keep-logs` and/or `--save-json` and/or `--save-test-json`
+    1. Navigate to performance test logs directory
+        ```bash
+        cd ./build/performance_test/
+        ```
+    2. Log Directory Structure is hierarchical with each run of the `performance_test.py` reporting into a timestamped directory where it includes the full performance report as well as a directory containing output from each test type run (here, `performance_test_basic.py`) and each individual test run outputs into a timestamped directory that may contain block data loggs and transaction generator logs as well as the test's basic report.  An example directory structure could look like:
```

Comment:
"An example directory structure could look like:" -> "An example directory structure follows:"

Examples are understood to be similar but not identical to what the user will see.  No need for further qualifiers.

Reply:
Addressed: [b641304](https://github.com/AntelopeIO/leap/pull/352/commits/b641304b5c62607aaac45e49a9d1de9cf701bba6)